historical_validation_code/evaluate_output_from_HV.R

```diff
@@ -186,6 +187,25 @@ n_seq_country_lineage<- clean_global_df %>% group_by(country, lineage, reference
 # Join with r_summary
 r_summary <- r_summary %>% left_join(n_seq_country_lineage, by = c('reference_date', 'country', 'lineage'))
 
+
+# Get Jensen-Shannon Divergence between the BA.5 global fitness advantage distributions
+BA.5_distrib <- mu_distrib %>% filter(lineage == 'BA.5')
+ref_date_1 <- BA.5_distrib %>% filter(reference_date == '2022-04-30') %>% pull(global_transmission_advantage)
+ref_date_2 <- BA.5_distrib %>% filter(reference_date == '2022-05-16') %>% pull(global_transmission_advantage)
+ref_date_3 <- BA.5_distrib %>% filter(reference_date == '2022-05-27') %>% pull(global_transmission_advantage)
+ref_date_4 <- BA.5_distrib %>% filter(reference_date == '2022-06-04') %>% pull(global_transmission_advantage)
+ref_date_5 <- BA.5_distrib %>% filter(reference_date == '2022-06-27') %>% pull(global_transmission_advantage)
+x <- rbind(ref_date_1, ref_date_2, ref_date_3, ref_date_4, ref_date_5) # These are not probability distributions, they are posterior distributions of the value of mu_hierarchical
```

Comment:
I think the posterior distribution of \mu is a probability distribution? It's p(\mu | X) where X is the data. But, I _think_ you're still right that we should use `est.prop = empirical` because we want to compute the relative frequency each value in the posterior predictive samples is observed.

Reply:
Yes sorry-- it is a probability distribution but we want the probability of that probability (so sum over all p(\mu|X)). It's explained here sorta... https://search.r-project.org/CRAN/refmans/philentropy/html/JSD.html

Reply:
The example is with counts vs frequencies, and in our case we have counts of mu values